### PR TITLE
[fix] #1698: Ensure client_cli is working and add documentation.

### LINKED ISSUE #1698: Ensure Client CLI is working
Make sure that the Client CLI is working for the preview release 1

### PR DESCRIPTION
**Pull request by appetrosyan**
### Issue
Closes #1698 

### Benefits

`client_cli` works. 

### Possible Drawbacks

None.

### CODE REVIEW

**Review by mversic — APPROVED** [review]

File: configs/client_cli/config.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "TORII_API_URL": "127.0.0.1:8080",
+  "TORII_API_URL": "http://127.0.0.1:8080",
   "ACCOUNT_ID": {
-    "name": "root",
-    "domain_name": "global"
+    "name": "alice",
+    "domain_name": "wonderland"
   },
   "PUBLIC_KEY": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
```
Comment (mversic) [inline configs/client_cli/config.json:7]: is public key in multihash format?

File: client_cli/src/main.rs [hunk 2]
```diff
@@ -115,41 +122,64 @@ fn main() -> Result<()> {
         config: Configuration(config),
         subcommand,
     } = Args::from_args();
-
-    println!("Value for config: {:?}", &config);
-    subcommand
-        .run(&config)
-        .wrap_err("Failed to run subcommand")?;
+    println!("Iroha Client CLI: build v0.0.1 [release]");
```
[the remark this answers is not in the document]
Comment (appetrosyan) [reply client_cli/src/main.rs:125]: We could, but it's too much of  a hassle and not strictly necessary.

File: configs/client_cli/config.json (hunk 1 above)
Comment (appetrosyan) [reply configs/client_cli/config.json:7]: Yes.

Comment (mversic) [reply configs/client_cli/config.json:7]: could you mention it somewhere in the doc?

**Review by eadventurous — APPROVED** [review]

File: configs/client_cli/config.json (hunk 1 above)
Comment (appetrosyan) [reply configs/client_cli/config.json:7]: I will add it to the `contributing` PR.

**Pull request merged**